ansible/plugins/lookup/hashivault.py

```diff
@@ -48,6 +48,10 @@ def _get_params(self, terms, kwargs):
         }
         authtype = os.getenv('VAULT_AUTHTYPE', 'token')
         params['authtype'] = authtype
+        params['ca_cert'] = os.getenv('VAULT_CACERT')
+        params['ca_path'] = os.getenv('VAULT_CAPATH')
```

Comment:
Up a couple lines we are setting params['verify'] to VAULT_CACERT or VAULT_CAPATH.  I forget if this is used at the requests level or urllib, but do you know the impact of this over that?  Seems like maybe this would work without the above two lines.